File: back/src/main/java/org/bliz/trivial/controller/model/QuestionsPerGame.java
package org.bliz.trivial.controller.model;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class QuestionsPerGame implements Serializable {

	private static final long serialVersionUID = -6680763609624970502L;

	private QuestionsPerGamePK id;

	private Integer isCorrect;

	private Integer isPlayed;

	private Game game;

	private Question question;

	private User user;
}

File: front/src/app/models/questionsPerGame.ts
import { User } from "./user";
import { Game } from "./game";
import { Question } from "./question";
import { QuestionsPerGamePK } from "./questionsPerGamePK";

export class QuestionsPerGame {

    id: QuestionsPerGamePK;
    isCorrect: number;
    isPlayed: number;
    game: Game;
    question: Question;
    user: User;
    // questionsPerGames: Array;

    constructor(id: QuestionsPerGamePK, isCorrect: number, isPlayed: number, game: Game, question: Question, user: User) {
        this.id = id;
        this.isCorrect = isCorrect;
        this.isPlayed = isPlayed;
        this.game = game;
        this.question = question;
        this.user = user;
        // this.questionsPerGames = questionsPerGames;
    }
}